### PR TITLE
Minor updates to Installing on Google Compute Engine topic

### PR DESCRIPTION
**Pull request by qahmed1998**
This PR just cleanups up some minor issues in "Installing on Google Compute Engine topic".

### CODE REVIEW

**Comment by qahmed1998** [comment]
@miq-bot assign @Fryguy

**Comment by agrare** [comment]
Hey @qahmed1998 could you add a more descriptive title and description about what this change is doing?

**Comment by qahmed1998** [comment]
@miq-bot add_labels radjabov/yes?

File: _includes/initial-login-changepw.md [hunk 1]
```diff
@@ -1,8 +1,7 @@
 Change your password to ensure more private and secure access to
 {{ site.data.product.title_short }}.
 
-1.  Browse to the URL for the login screen. (<https://xx.xx.xx.xx> on
-    the virtual machine instance)
+1.  Browse to the URL for the login screen.
```
Comment (qahmed1998) [inline _includes/initial-login-changepw.md:4]: @Fryguy how can we improve this line?

Comment (Fryguy) [reply _includes/initial-login-changepw.md:4]: The parenthesis shows an example, so maybe adding an `e.g.`? I'm not sure what was wrong with it originally.

Comment (agrare) [reply _includes/initial-login-changepw.md:4]: Yeah I think the original was okay, maybe something to say "get the ipaddress or hostname from the VM details page"

**Comment by agrare** [comment]
@qahmed1998 FYI it looks like these changes are all to includes which are in multiple topics not just google compute engine

File: _includes/initial-login-changepw.md (hunk 1 above)
Comment (qahmed1998) [reply _includes/initial-login-changepw.md:4]: I meant like in terms of navigation, how does a user get the URL?

Comment (agrare) [reply _includes/initial-login-changepw.md:4]: It is going to be different on every platform (vmware, amazon, google, etc...) but all of them will show you the IP address and/or dns name of the VM in some way.

Comment (qahmed1998) [reply _includes/initial-login-changepw.md:4]: Hmm for example this doc is for GCP, I am still not sure how exactly user will get the URL for the login screen.

Comment (agrare) [reply _includes/initial-login-changepw.md:4]: This doc isn't for GCP, it is included into all of the "installing_on_*" docs,
```
$ grep -r initial-login-changepw
installing_on_scvmm/index.md:{% include initial-login-changepw.md %}
installing_on_vmware_vsphere/index.md:{% include initial-login-changepw.md %}
installing_on_microsoft_azure/index.md:{% include initial-login-changepw.md %}
installing_on_google_compute_engine/index.md:{% include initial-login-changepw.md %}
installing_on_red_hat_virtualization/index.md:{% include initial-login-changepw.md %}
grep: .git/index: binary file matches
installing_on_red_hat_enterprise_linux_openstack_platform/index.md:{% include initial-login-changepw.md %}
```
Which was the point of my comment :point_down: https://github.com/ManageIQ/manageiq-documentation/pull/1805#issuecomment-2289091895

Comment (agrare) [reply _includes/initial-login-changepw.md:4]: IMO I think it is reasonable to assume that a user will know how to get the IP address of the VM that they just deployed on whatever platform they're using.

Comment (qahmed1998) [reply _includes/initial-login-changepw.md:4]: ok thanks Adam.

**Comment by qahmed1998** [comment]
Hi @agrare @Fryguy Please approve/merge if all looks good.

**Review by agrare — APPROVED** [review]

**Comment by Fryguy** [comment]
Backported to `radjabov` in commit 4eb94c9964183368a32c572242a9570d7c8a54b8.

```text
commit 4eb94c9964183368a32c572242a9570d7c8a54b8
Author: Adam Grare <adam@grare.com>
Date:   Tue Aug 20 10:35:48 2024 -0400

    Merge pull request #1805 from qahmed1998/QA-minor-changes-gcp
    
    Minor updates to Installing on Google Compute Engine topic
    
    (cherry picked from commit c64feef8832df1d360790ed455e17d01d3b28145)
```

**Pull request merged**